Java_fibonacci.java

```diff
@@ -1,11 +1,9 @@
-package milestone6;
-
 import java.util.Scanner;
+import java.math.BigInteger;
 
-public class JavaFibonacci {
+public class Java_fibonacci {
```

Comment:
Codacy found an issue: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6357937)